flake.nix

```diff
@@ -52,7 +52,6 @@
           code = import ./code/cardano-deposit-wallet.nix {
             inherit system;
             inherit (inputs) nixpkgs haskellNix iohkNix CHaP flake-utils;
-            rewrite-libs = rewrite-libs.packages.default;
```

Comment:
Wait, don't we lose the fact that the shared libraries on macOS are relative to the executable in this way? 🤔 That would be detrimental.

https://github.com/cardano-foundation/cardano-deposit-wallet/blob/60ffdb861fd29d53f7bdba2fc40e72ed69366235/nix/macos-artifacts.nix#L32-L37

Reply:
Why did I remove it ?

Reply:
Because it's unused there, rewrite-libs is used only in  packaging

Reply:
Because it's unused there, rewrite-libs is used only in  packaging

Reply:
Oooh, I see. Sorry about the noise. 👍